src/components/board/BoardCard.jsx

```diff
@@ -1,4 +1,19 @@
-const BoardCard = () => {
-  return <div></div>;
+import Image from "next/image";
+import Link from "next/link";
+
+const BoardCard = ({ image, name, position, linkedIn }) => {
+  return (
+    <div className="space-y-3">
+      <div className="flex justify-center">
+        <Link href={linkedIn} className="inline-block">
+          <Image src={image} alt="picture of person" />
+        </Link>
+      </div>
+      <div className="space-y-0.5">
+        <p className="text-4xl flex justify-center">{name}</p>
```

Comment:
wrong font family, make sure a bolded effect is added probably.. font-medium? you'll have to check